xml/cap_depl_service_broker_sidecar.xml

```diff
@@ -165,14 +165,14 @@
    -----END CERTIFICATE-----
    
    # UAA cert
-   UAA_CA_CERT: |
-    -----BEGIN CERTIFICATE-----
```

Comment:
I don't think the spaces need to be removed on the lines with the cert's contents if we're going for 2 spaces per indent level. If that's the case, other areas will need to be updated because we have varying levels of indentation throughout the docs.

Reply:
We do need to remove the space before `UAA_CA_CERT:` (to line up with the first element after the `env:`, which is `SERVICE_MYSQL_HOST:` here).

There can be any number of spaces in front of the _contents_ of `UAA_CA_CERT`, as long as we're consistent about it.

Reply:
2 spaces per indent level sounds good. I'll review all the example yaml files in the Deployment Guide.